### PR TITLE
Dashboard 3.0 Devops Update

### LINKED ISSUE #143: add .eslint file
So we can silence the warnings we don't like.

### LINKED ISSUE #144: Add flake8 config

### LINKED ISSUE #146: Delete beta branch (and associated git actions)
We don't want it anymore.

### LINKED ISSUE #147: Refine deploy script
git fetch can't be unshallow when not run by an action.
also, beta should be default, prod only when flag is passed?

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Changes:
- Added `.eslintrc.js` (fixes #143)
- Added `.flake8` (fixes #144)
- Removed `deploy_beta.yml` GitHub Action (fixes #146)
- Modified `deploy.sh` to take command line arguments to specify production and CI runs (fixes #147)
- Updated `deploy.yml` GitHub Action to be compatible with `deploy.sh` changes
- Updated `package.json` and `README.md`

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -41,4 +41,4 @@ jobs:
         run: |
           pip install pipenv
           npm install
-          bash deploy.sh
+          bash deploy.sh -p -c
```
Comment (mathcolo) [inline .github/workflows/deploy.yml:44]: Out of curiosity does this work as `-pc` in addition to `-p -c`?

**Review by mathcolo — APPROVED** [review]
noice! 😃

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (nathan-weinberg) [reply .github/workflows/deploy.yml:44]: A quick test shows it does!

Comment (mathcolo) [reply .github/workflows/deploy.yml:44]: 👀 nice!!!

File: deploy.sh [hunk 2]
```diff
@@ -1,25 +1,55 @@
...
-    GIT_ID=`git describe --always --dirty --abbrev=10`
-    echo "Deploying git commit id $GIT_ID"
+# Fetch repository tags
+# Run unshallow if deploying in CI
+
+if $CI; then
+    git fetch --unshallow --tags
 else
+    git fetch --tags
+fi
```
Comment (austinjpaul) [inline deploy.sh:42]: shouldn't need the fetch here, since it's deploying whatever's in the local directory.

**Review by austinjpaul — APPROVED** [review]
looks great!

**Pull request merged**